### DISCUSSION TITLE
How to upload a simple file

### DISCUSSION THREAD

**Discussion by wilmaracl** [community/community#22598] [New to GitHub]
<p>I created an GitHub account and want to populate it. I am using Linux Mint.</p>
<p>I had access my repository  wilmaracl/Computacional-II.git.</p>
<p>I choose    <strong>&lt;&gt;Code </strong>     but the    <strong>Upload File</strong>     option wasn’t there. On that page I found:</p>
<p><strong>Quick setup</strong> — if you’ve done this kind of thing before</p>
<p><strong>or</strong><br>
   HTTPS    SSH     <a href="mailto:git@github.com">git@github.com</a>:wilmaracl/Computacional-II.git</p>
<p>We recommend every repository include a <a href="https://github.com/wilmaracl/Computacional-II/new/master?readme=1" rel="nofollow noopener">README</a>, <a href="https://github.com/wilmaracl/Computacional-II/new/master?filename=LICENSE.md" rel="nofollow noopener">LICENSE</a>, and <a href="https://github.com/wilmaracl/Computacional-II/new/master?filename=.gitignore" rel="nofollow noopener">.gitignore</a>.</p>
<p>…or create a new repository on the command line</p>
<pre><code>echo "# Computacional-II" \&gt;\&gt; README.mdgit initgit add README.mdgit commit -m "first commit"git remote add origin git@github.com:wilmaracl/Computacional-II.gitgit push -u origin master
</code></pre>
<p>…or push an existing repository from the command line</p>
<pre><code>git remote add origin git@github.com:wilmaracl/Computacional-II.gitgit push -u origin master
</code></pre>
<p>…or import code from another repository</p>
<p>You can initialize this repository with code from a Subversion, Mercurial, or TFS project.</p>
<p>I’ve done something wrong? There are things that I didn’t do?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi and welcome <a class="mention" href="/wilmaracl">@wilmaracl</a>,</p>
<p>You have created an empty repository. This is handy for when you already have a functioning Git repository offline and just want to “import”/push it to GitHub. However, you don’t have that.</p>
<p>To accomplish what you would like, you can do two things:</p>
<ol>
<li>Create an empty repository like you did here, then create a local repository, create the files in there and push that local repository</li>
<li>Check the box “Initialize this repository with a README.md” upon creating a repository (a repository with a file called README.md will be created and you will see the regular view you are used to when visiting the repository)</li>
</ol>
<p>As for this specific case, to accomplish what you need you can do the following things:</p>
<ul>
<li>Using solution 1 is still possible, so if you would like that go ahead and do that</li>
<li>If you would like to go with the second solution, delete the repository and create a new one with the box checked</li>
</ul>

**Comment by wilmaracl** [comment]
<p>Thank you Mark.</p>
<p>I did the first option and it works.</p>
<p>Now I already have my first Python (Serie RLC circuit) on my repository Computacional-II</p>
<p>Wilmar - Electrical Engineer - Brazil - Goiás - Goiânia</p>
